src/pkg/packager/prepare.go

```diff
@@ -356,6 +377,28 @@ func (p *Packager) processUnstructuredImages(resource *unstructured.Unstructured
 	return matchedImages, maybeImages, nil
 }
 
+func (p *Packager) findWhyResources(resources []*unstructured.Unstructured, whyImage, componentName, resourceName string, isChart bool) []string {
+	output := []string{}
+	for _, resource := range resources {
+		bytes, _ := resource.MarshalJSON()
+		json := string(bytes)
+		resourceTypeKey := "manifest"
+		if isChart {
+			resourceTypeKey = "chart"
+		}
+
+		var imageFuzzyCheck = regexp.MustCompile(`(?mi)["|=]([a-z0-9\-.\/:]+:[\w.\-]*[a-z\.\-][\w.\-]*)"`)
+		matches := imageFuzzyCheck.FindAllStringSubmatch(json, -1)
+		for _, group := range matches {
+			if whyImage == group[1] {
+				yamlResource, _ := yaml.Marshal(resource.Object)
+				output = append(output, fmt.Sprintf("component: %s\n%s: %s\nresource: %s\n\n%s\n", componentName, resourceTypeKey, resourceName, resource.GetName(), string(yamlResource)))
```

Comment:
We shouldn't need `resource.GetName()` here because the name of the YAML manifest will be visible in the console output